docs/context.rst

```diff
@@ -0,0 +1,64 @@
+Context & Breadcrumbs
+=====================
+
+The Java SDK implements the idea of a "context" to support attaching additional
+information to a specific user request, such as breadcrumbs.
```

Comment:
"specific user request" is a bit over-specific and limiting, it would be better to go with something more generic here.